public_html/wp-content/plugins/wordcamp-payments/includes/sponsor-invoice.php

```diff
@@ -419,6 +419,13 @@ function save_invoice( $post_id, $post ) {
 			update_post_meta( $post_id, $meta_key, $value );
 		}
 	}
+
+	if ( 'wcbsi_approved' === $post->post_status ) {
+		$invoice_sent_at = get_post_meta( $post_id, 'Sent at', true );
+		if ( empty( $invoice_sent_at ) ) {
+			update_post_meta( $post_id, 'Sent at', true );
```

Comment:
I think the value here should be `time()` instead of `true`?

Reply:
fixed